share/cmake/cccl/cccl-config.cmake

```diff
@@ -0,0 +1,59 @@
+#
+# find_package(CCCL) config file.
+#
+# Imports the Thrust, CUB, and libcudacxx components of the NVIDIA
+# CUDA/C++ Core Libraries.
+
+unset(cccl_version) # TODO
+
+set(cccl_cmake_dir "${CMAKE_CURRENT_LIST_DIR}")
+
+if (${CMAKE_FIND_PACKAGE_NAME}_FIND_QUIETLY)
+  set(cccl_quiet_flag "QUIET")
+else()
+  set(cccl_quiet_flag "")
+endif()
+
+foreach(component ${${CMAKE_FIND_PACKAGE_NAME}_FIND_COMPONENTS})
```

Comment:
I would expect no components is equal to all components, and not zero components and therefore nothing is brought int

Reply:
Agreed. I will check my tests to make sure that this is what's going on.

Reply:
Found the bug -- fixed it. All is well now.